tools/restatectl/src/commands/partition/list.rs

```diff
@@ -79,27 +107,85 @@ pub async fn list_partitions(
                     let host_node =
                         GenerationalNodeId::new(host.id, host.generation.expect("generation"));
                     let details = PartitionListEntry { host_node, status };
-                    partitions.entry(partition_id).or_default().push(details);
+                    partitions.push((partition_id, details));
                 }
             }
         }
     }
+
     // Show information organized by partition and node
     let mut partitions_table = Table::new_styled();
     partitions_table.set_styled_header(vec![
         "P-ID",
         "NODE",
         "MODE",
         "STATUS",
-        "APPLIED",
-        "PERSISTED",
         "LEADER",
         "EPOCH",
+        "SEQUENCER",
```

Comment:
What's the meaning of this column?

Reply:
Added a comment and subcommand help - it's a feature request to support seeing PP leader and the underlying loglet sequencer in the same table :-)

Reply:
I've expanded the about doc that gets printed under `--help` - hope that covers it!